extractors/xtract_netcdf.py

```diff
@@ -0,0 +1,102 @@
+from extractors.extractor import Extractor
+
+
+class NetCDFExtractor(Extractor):
+
+    def __init__(self):
+        super().__init__(extr_id=None,
+                         func_id="",  # TODO: Fill in once func is registered
+                         extr_name="xtract-netcdf",
+                         store_type="ecr",
+                         store_url="039706667969.dkr.ecr.us-east-1.amazonaws.com/xtract-netcdf:lateset")
+        super().set_extr_func(netcdf_extract)
+
+
+def netcdf_extract(event):
+    """Extract metadata from netcdf data.
+
+    Parameters
+    ----------
+    event : dict
+        A dict describing the data and credentials to act on
+
+    Returns
+    -------
+    dict : The resulting metadata and timers
+    """
+    import sys
+    import time
+    import os
+
+    from xtract_sdk.downloaders.google_drive import GoogleDriveDownloader
+
+    t0 = time.time()
+
+    sys.path.insert(1, '/')
+    import xtract_netcdf_main
+    # from exceptions import RemoteExceptionWrapper, HttpsDownloadTimeout, ExtractorError, PetrelRetrievalError
+
+    def min_hash(fpath):
```

Comment:
You don't actually need the minhash function in here. That was a weird 'after the fact' add-on for another collaboration I'm working on with another student.